statvar_imports/india_ndap/ndap/download_script.py

```diff
@@ -0,0 +1,43 @@
+import sys
+import pandas as pd
+import config
+import os
+from absl import logging, app
+
+_SCRIPT_PATH = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(_SCRIPT_PATH, '../../../util/')) 
+from download_util_script import _retry_method
+
+
+def main(_):
+    all_data = []
+
+    for page_num in range(1, 3):  
+        api_url = f"{config.url}&pageno={page_num}"
+        response = _retry_method(api_url, None,  3, 5,2)
+
+        response_data = response.json()
+
+        if response_data:
+            keys = list(response_data.values())[5]  
+            
+            for i in keys:
+                a = a=i['StateName'],i['Year'].split(",")[-1].strip(),i['GENDER'],i['I7375_4']['TotalPopulationWeight'],i['Year'].split(",")[-1].strip(),i['Year']
```

Comment:
is I7375_4 a hardcoded indicator code ? Can you add more context as comment ?

Reply:
The I7375_4 is the table ID and is present inside the table as well.
Added a comment